### DISCUSSION TITLE
Is a 'view component source' link useful

### DISCUSSION THREAD

**Discussion by nicolethoen** [patternfly/patternfly#5514] [General]
On each component page, would it be useful to add a link to view the source code for the component?

**Comment by gitdallas** [comment]
I've went and looked at the code myself when I wasn't sure about something, so I think this would make sense. Also conveys the whole "open source" aspect better and might encourage PF consumers to participate more.

**Comment by KKoukiou** [comment]
I think it's not interesting for most of the users, and for those few for whom it is, it's straightforward to navigate in the react-core. So my input here would be 'let's not add such link'
